packages/paste-core/components/help-text/stories/HelpText.storiess.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {HelpText, HelpTextVariants} from '../src';
+
+/* eslint-disable-next-line import/no-default-export */
+export default {
+  title: 'Components/HelpText',
+  component: HelpText,
+};
+
+const Template: React.FC<{variant: HelpTextVariants}> = ({variant}) => (
```

Comment:
TODO: Not sold on this API, need to think some more on how to make it so that we can use these stories in docs and elsewhere

Reply:
You can use the named Exported components in other places. We re-use them for tests so storybook has some additional testing for us.